build.gradle

```diff
@@ -1289,7 +1267,7 @@ project(':metadata') {
     }
     test {
       java {
-        srcDirs = ["src/generated/java", "src/test/java"]
+        srcDirs = ["src/test/java"]
```

Comment:
Why did you need to remove the generated sources? Can't we just exclude these packages from the coverage report?

Reply:
it's not exclusion, it's deduplication 
so it's not strictly conflicts with coverage, but since the src/generated/java is already there in sourceSets.main.java it is unnecessary to repeat it in test.java sourceset as well.

Sonar plugin was the one which wasn't working until these were cleared out.